.cirrus.yml

```
@@ -1,18 +1,36 @@
-container:
-  image: gcr.io/flutter-cirrus/build-flutter-image:latest
+# CIRRUS CONFIGURATION FILE
 
+env:
+  # We change Flutter's directory to include a space in its name (see
+  # $CIRRUS_WORKING_DIR) so that we constantly test path names with
+  # spaces in them. The FLUTTER_SDK_PATH_WITH_SPACE variable must
+  # therefore have a space in it.
+  FLUTTER_SDK_PATH_WITH_SPACE: "flutter sdk"
+  GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
+  BOT: "true"
+
+clean_commit_message_script:
+  # Unsetting CIRRUS_CHANGE_MESSAGE and CIRRUS_COMMIT_MESSAGE as they
+  # might include non-ASCII characters which makes Gradle crash.
+  # See: https://github.com/flutter/flutter/issues/24935
+  # TODO(amirha): remove once we've migrated to newer Gradle
+  - echo "$CIRRUS_CHANGE_MESSAGE" > /tmp/cirrus_change_message.txt
+  - echo "$CIRRUS_COMMIT_MESSAGE" > /tmp/cirrus_commit_message.txt
+  - export CIRRUS_CHANGE_MESSAGE=""
+  - export CIRRUS_COMMIT_MESSAGE=""
+
+# LINUX SHARDS
 task:
+  container:
+    image: gcr.io/flutter-cirrus/build-flutter-image:latest
+    cpu: 1 # 0.1-8 without compute credits, 0.1-30 with (yes, you can go fractional)
```

You can put `$CPU` here:

```suggestion
    cpu: $CPU # 0.1-8 without compute credits, 0.1-30 with (yes, you can go fractional)
```

---

nice!

---

@fkorotkov this didn't seem to work? Maybe I did it wrong.

---

never mind, i understand